sentry/src/main/java/io/sentry/SentryMeasurementUnit.java

```diff
@@ -0,0 +1,40 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * <a href="https://getsentry.github.io/relay/relay_metrics/enum.DurationUnit.html">Develop Docs</a>
+ */
+public enum SentryMeasurementUnit {
+  /** Nanosecond (`"nanosecond"`), 10^-9 seconds. */
+  NANOSECOND,
+
+  /** Microsecond (`"microsecond"`), 10^-6 seconds. */
+  MICROSECOND,
+
+  /** Millisecond (`"millisecond"`), 10^-3 seconds. */
+  MILLISECOND,
+
+  /** Full second (`"second"`). */
+  SECOND,
+
+  /** Minute (`"minute"`), 60 seconds. */
+  MINUTE,
+
+  /** Hour (`"hour"`), 3600 seconds. */
+  HOUR,
+
+  /** Day (`"day"`), 86,400 seconds. */
+  DAY,
+
+  /** Week (`"week"`), 604,800 seconds. */
+  WEEK,
+
+  /** Untyped value without a unit. */
+  NONE;
+
+  public @NotNull String apiName() {
+    return name().toLowerCase(Locale.ROOT);
+  }
```

Comment:
`m`: Does this have to be public? I don't think you need this as a user.

Reply:
it's not for the user but for the sentry-android module where we use it directly

Reply:
You can annotate with @internal if needed